app/presenters/instance_presenter.rb

```diff
@@ -13,6 +13,10 @@ class InstancePresenter
     to: Setting
   )
 
+  def active_count(timespan: Time.zone.now - 1.month..Time.zone.now)
+    Status.select('distinct (account_id)').where(local: true, created_at: timespan).count
```

Comment:
users that log on and don't post are still active.

Reply:
I wasn't sure exactly how to determine that since a user could stay logged in for a long period of time?  I welcome suggestions from anyone who knows the schema better than me.

Reply:
That's why we track statistics explicitly in `UserTrackingConcern`

Reply:
Sorry, `ActivityTracker`